subsys/net/lib/lwm2m_client_utils/location/location_event_handler.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <nrf_modem_gnss.h>
+
+#include <net/lwm2m_path.h>
+#include <net/lwm2m_client_utils_location.h>
+#include <net/lwm2m_client_utils.h>
+
+#define MODULE lwm2m_location_event_handler
```

Comment:
Do we need this define? Could use the module name directly in `LOG_MODULE_REGISTER()`